src/univariate/continuous/normal.jl

```diff
@@ -35,6 +35,11 @@ entropy(d::Normal) = 0.5 * (log2π + 1.0) + log(d.σ)
 
 @_delegate_statsfuns Normal norm μ σ
 
+### Added by scidom: start
+logpdf(d::Normal, x::Real) = -(log(2*π)/2+log(d.σ)+abs2((x-d.μ)/d.σ)/2)
+logpdf(d::Normal, x::Number) = -(log(2*π)/2+log(d.σ)+abs2((x-d.μ)/d.σ)/2)
```

Comment:
The compiler complained I should define the case of real input argument first... and then I can't define the second function by invoking the first, as x must be passed as a number. How could I deal with this?